### PR TITLE
[AV2-273]/new upload video preview issue

### PR DESCRIPTION
**Pull request by ronakamani**

### CODE REVIEW

File: src/app/questions/file/file-display/file-display.component.ts [hunk 1]
```diff
@@ -25,6 +27,7 @@ export class FileDisplayComponent implements OnInit, OnChanges {
   private resetUILogic() {
     this.virusDetection = {};
     this.quarantine = {};
+    this.video?.nativeElement.load();
```
Comment (shawnm0705) [inline src/app/questions/file/file-display/file-display.component.ts]: is the question mark intended?

File: src/app/questions/file/file-display/file-display.component.ts [hunk 2]
```diff
@@ -13,8 +13,10 @@ export class FileDisplayComponent implements OnInit, OnChanges {
 
   @Input() fileType = 'any';
   @Input() file: any;
+  @ViewChild('video')
+  public video: ElementRef;
```
Comment (shawnm0705) [inline src/app/questions/file/file-display/file-display.component.ts]: better to put those into one line

File: src/app/questions/file/file-display/file-display.component.html [hunk 3]
```diff
@@ -6,13 +6,8 @@
     </ng-container>
 
     <ng-container *ngSwitchCase="'video'">
-      <video controls>
-        <ng-container *ngIf="file.transcoded">
-          <source [src]="file.transcoded + '.mp4#t=0.5'" type="video/mp4">
-          <source [src]="file.transcoded + '.webm'" type="video/webm">
-          <source [src]="file.transcoded + '.ts'">
-        </ng-container>
```
Comment (shawnm0705) [inline src/app/questions/file/file-display/file-display.component.html]: no need to remove the transcoded ones

File: src/app/questions/file/file-display/file-display.component.ts (hunk 1 above)
Comment (ronakamani) [reply src/app/questions/file/file-display/file-display.component.ts]: yes, if I remove that I'll get this error:
ERROR TypeError: Cannot read property 'nativeElement' of undefined

**Comment by ronakamani** [comment]
Hi @shawnm0705 , 
the test errors which left are related to utilService like before.

**Comment by Barry-Chen-Intersective** [comment]
Your global Angular CLI version (9.1.0) is greater than your local
--
181 | version (9.0.5). The local Angular CLI version is used.
182 |  
183 | To disable this warning use "ng config -g cli.warnings.versionMismatch false".
184 | Linting "app"...
185 |  
186 | ERROR: /codebuild/output/src682589293/src/github.com/intersective/practera-app-v2/src/app/questions/file/file-display/file-display.component.spec.ts:188:1 - trailing whitespace
187 |  
188 | Lint errors found in the listed files.
189 | Linting "app-e2e"...
190 | All files pass linting.

**Pull request merged**